client/scala/examples/README.md

```diff
@@ -1,3 +1,11 @@
-To build the example Armada client programs, copy the jar file that was generated (e.g.
-scala-armada-client_2.13-0.1.0-SNAPSHOT.jar) from ../scala-armada-client/target/scala-2.13/
-into the `lib` directory, then run `sbt compile` here.
+To build the example Armada client programs, build the Scala Armada client first:
+```
+cd client/scala/scala-armada-client/
+sbt publishLocal
```

Comment:
either we use
```suggestion
sbt publishM2
```
here or remove the `resolvers += Resolver.mavenLocal` from the `examples/build.sbt` below.